src/Mvc/Mvc.Razor.RuntimeCompilation/src/PACKAGE.md

```diff
@@ -0,0 +1,75 @@
+## About
+
+Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation is a NuGet package designed to provide runtime compilation support for Razor views in ASP.NET Core MVC applications. This package enables developers to modify and update Razor views without needing to restart the application, facilitating a more dynamic development experience.
+
+## Key Features
```

Comment:
Thanks @lyndaidaii.
Can we also include a note about things that aren't supported, or at least state that some things aren't supported, and can be found in the documentation: https://learn.microsoft.com/en-us/aspnet/core/mvc/views/view-compilation?view=aspnetcore-8.0&tabs=visual-studio